Remotion/Data/DomainObjects/Utilities/ObjectIDConverter.cs

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Utilities;
+
+public class ObjectIDConverter : TypeConverter
+{
+  private readonly Type _stringType = typeof(string);
```

Comment:
Not sure why `typeof(string)` is in a field. I don't think it adds any benefit and can be inlined instead. If you really want to use a field to remove the duplication, it should be `static` instead. But `typeof(...)` is a special case where the compiler can generate efficient code anyways so a field is not needed anyway. For example:

```
public Type M() => typeof(string);
```

compiles down to a constant move:

```
C.M()
    L0000: mov eax, 0xa000014
    L0005: ret
```